pkgs/net-test/tests/net/server.rkt

```diff
@@ -46,9 +46,9 @@
          (keyword-apply
           start-server
           kws kw-args
-          "127.0.0.1" 0 handler-proc
-          args
-          #:listen-proc (make-tcp-listen-proc port-ch)))
+          ((make-tcp-listen-proc port-ch) 0 4 #t "127.0.0.1")
```

Comment:
This backlog value makes some of the tests fail spuriously on my machine. Let's bump it to 512 since these tests can launch many concurrent clients very quickly.